src/index.html

```diff
@@ -9,6 +12,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
```

Comment:
```suggestion
    <header class="header">
```
Don't leave empty lines between parent and child elements